pkg/addons/addons.go

```diff
@@ -279,6 +281,19 @@ func EnableOrDisableAddon(cc *config.ClusterConfig, name string, val string) err
 		return errors.Wrap(err, "command runner")
 	}
 
+	crName := cc.KubernetesConfig.ContainerRuntime
+	cr, err := cruntime.New(cruntime.Config{Type: crName, Runner: runner})
+	if err != nil {
+		return errors.Wrap(err, "container runtime")
+	}
+	runtimePaused, err := cluster.CheckIfPaused(cr, []string{"kube-system"})
+	if err != nil {
+		return errors.Wrap(err, "check paused")
+	}
+	if runtimePaused {
+		out.Styled(style.Shrug, `Can't enable addon on a paused cluster, please unpause the cluster firstly.`)
+		return errors.New("Can't enable addon on a paused cluster, please unpause the cluster firstly.")
```

Comment:
Since the exact same string is being used twice it should be assigned to a variable and then pass the variable to each function. That way if we update the message in the future we don't have to update two strings.

Reply:
Thanks for reviewing! Updated the code to use a shared string variable.